content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -51,6 +51,14 @@ Use the following environment variables to configure formats for reading and wri
 : Specifies trace context propagation formats for both extraction and injection (comma-separated list). Lowest precedence; ignored if any other Datadog trace context propagation environment variable is set.<br>
 **Note**: Only use this configuration when migrating an application from the OpenTelemetry SDK to the Datadog SDK. For more information on this configuration and other OpenTelemetry environment variables, see [Using OpenTelemetry Environment Variables with Datadog SDKs][9].
 
+`DD_TRACE_PROPAGATION_BEHAVIOR_EXTRACT`
+: Specifies how incoming distributed tracing headers should be handled at a service level. Accepted values are:<br>
```

Comment:
```suggestion
: Specifies how incoming distributed tracing headers should be handled. Accepted values are:<br>
```
Just a suggestion, but the extra words made it more confusing to me and I don't think they're necessary? Or maybe say "...on service entry spans" instead?

Reply:
Hmm the message I'm trying to convey is that this affects the entire application, whereas in the future I'd like to add support by _endpoint_ (e.g. /healthcheck vs /cart) . Do you think that's necessary to outline right now or we can omit it for now?